aws_lambda_builders/workflows/go_modules/workflow.py

```diff
@@ -33,8 +33,14 @@ def __init__(
         output_path = osutils.joinpath(artifacts_dir, handler)
 
         builder = GoModulesBuilder(
-            osutils, binaries=self.binaries, mode=mode, architecture=self.architecture, trim_go_path=trim_go_path
+            osutils,
+            binaries=self.binaries,
+            mode=mode,
+            architecture=self.architecture,
+            trim_go_path=trim_go_path,
+            handler=handler,
```

Comment:
nit: just keeping the order in the constructor definition for easy readability
```suggestion
            osutils,
            binaries=self.binaries,
            handler=handler,
            mode=mode,
            architecture=self.architecture,
            trim_go_path=trim_go_path,
```

Reply:
right, done!